go.mod

```diff
@@ -577,13 +577,17 @@ require (
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
 
+require (
+	github.com/google/dnsmasq_exporter v0.2.1-0.20230620100026-44b14480804a
+	github.com/hashicorp/golang-lru/v2 v2.0.3
+	github.com/zeebo/assert v1.3.0
+)
```

Comment:
nit: is there a reason we've introduced this new library? Otherwise we an reuse testify/require and merge this so we don't have an extra require block to keep track of.

Reply:
Oh, my bad. I blame copilot.